substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -202,6 +202,10 @@ protected void onValueUpdate(EconomicMap<OptionKey<?>, Object> values, String ol
     public static OptionEnabledHandler<Boolean> imageLayerEnabledHandler;
     public static OptionEnabledHandler<Boolean> imageLayerCreateEnabledHandler;
 
+    @APIOption(name = "-javaagent", valueSeparator = ':')//
+    @Option(help = "Enable the given java agent in native image.", type = User, stability = OptionStability.EXPERIMENTAL)//
```

Comment:
Please specify the expected format here.

Reply:
Fixed